install/mysql/glpi-empty.sql

```diff
@@ -2719,6 +2719,7 @@ CREATE TABLE `glpi_entities` (
   `send_licenses_alert_before_delay` int NOT NULL DEFAULT '-2',
   `use_certificates_alert` int NOT NULL DEFAULT '-2',
   `send_certificates_alert_before_delay` int NOT NULL DEFAULT '-2',
+  `certificates_alert_repeat` int NOT NULL DEFAULT '-2',
```

Comment:
I had to read the code to be able to know whether this field was containing a boolean or an interval.
```suggestion
  `certificates_alert_repeat_interval` int NOT NULL DEFAULT '-2',
```

EDIT: I saw after my review that this name was inspired by `cartridges_alert_repeat`, but I still think that names should be clear enough to prevent any ambiguity.